Update docs (#2067)
kyma-project/control-plane: docs/kyma-environment-broker/03-03-runtime-operations.md

> **NOTE:** It's important to set lower timeouts for the Kyma installation in the Runtime Provisioner.

## Operations

> 

Reviewer: I think we need to describe what stage it is:
 - once stage is finished, won't be repeated (in opposite to step)
 - stage groups steps
do not describe here operation nor steps (it is described at the end of the doc)

   </details>
</div>

## Stages

> Operation defines stages and steps, which represents a part of work to be done.

Reviewer 2: ```suggestion
An operation defines stages and steps which represent the work you must do.
```

> The stage is a grouping unit for steps.

Reviewer 2: ```suggestion
A stage is a grouping unit for steps.
```

> Step is a part of the stage

Reviewer 2: ```suggestion
A step is a part of a stage.
```

> Operation can contain multiple stages, and stage can contain multiple steps.

Reviewer 2: ```suggestion
An operation can consist of multiple stages, and a stage can consist of multiple steps.
```

> We group the steps inside one stage when we have some sensitive data which we don't want to store in database, then we are keeping them temporary in memory and pass through steps.

Reviewer 2: ```suggestion
You group steps in a stage when you have some sensitive data which you don't want to store in database. In such a case you temporarily store the sensitive data in the memory and go through the steps.
```

> Once all steps in stage are executed with success, then stage is marked as finished and never repeat again, even if next one will fail.

Reviewer 2: ```suggestion
Once all the steps in a stage are successfully executed, the stage is marked as finished and never repeated again, even if the next one fails.
```

> If any of steps will fail in given stage, then whole stage will be repeated from beginning.

Reviewer 2: ```suggestion
If any steps fail at a given stage, the whole stage is repeated from the beginning.
```
